### PR TITLE
Fix case of types in `@inferred` docstring

### PR DESCRIPTION
**Pull request by jishnub**

### CODE REVIEW

File: stdlib/Test/src/Test.jl [hunk 1]
```diff
@@ -1722,11 +1722,11 @@ Int64
 
 julia> @code_warntype f(2)
 MethodInstance for f(::Int64)
-  from f(a) @ Main none:1
```
Comment (KristofferC) [inline stdlib/Test/src/Test.jl:1725]: I don't think this will pass the docstring anymore.

Comment (jishnub) [reply stdlib/Test/src/Test.jl:1725]: I'll revert this then

File: stdlib/Test/src/Test.jl [hunk 2]
```diff
@@ -1722,11 +1722,11 @@ Int64
...
+  from f(a) in Main at REPL[3]:1
 Arguments
   #self#::Core.Const(f)
   a::Int64
-Body::UNION{FLOAT64, INT64}
+Body::Union{Float64, Int64}
```
Comment (fredrikekre) [inline stdlib/Test/src/Test.jl:1729]: Julia uses uppercase instead of color when there is no color support, so this change will also break CI.

Comment (jishnub) [reply stdlib/Test/src/Test.jl:1729]: Oh, I see.

**Pull request closed without merging**